paasta_itests/paasta_deployd.feature

```diff
@@ -74,7 +74,7 @@ Feature: paasta-deployd deploys apps
       And we set the instance count in zookeeper for service "test-service" instance "main" to 3
      Then we should see "test-service.main" listed in marathon after 45 seconds
      Then we should see the number of instances become 3
-     When we set the instance count in zookeeper for service "test-service" instance "main" to 4
+     When we set the instance count in zookeeper for service "test-service" instance "main" to 5
```

Comment:
Can you add a comment explaining why it only goes to 4?

Reply:
Fixed